src/csm_testing/tests/sat_functional/bootprep/test_bootprep.py

```diff
@@ -439,36 +428,84 @@ def delete_cfs_configuration(cfs_config_name):
             logging.warning('Failed to delete CFS configuration "%s" '
                             'created by test: %s', cfs_config_name, err.stderr)
 
-    @staticmethod
-    def delete_all_cfs_configurations_matching_prefix(cfs_config_prefix):
+    @classmethod
+    def delete_all_cfs_configurations_matching_prefix(cls, cfs_config_prefix):
         """Find and delete all CFS configurations matching a prefix using the 'cray' CLI.
 
         This relies on the cray CLI being configured and authenticated on the system.
+
+        Args:
+            cfs_config_prefix (str): cfs configuration prefix to match
         """
         find_command = 'cray cfs v3 configurations list'
         found_configurations = []
         try:
             proc = subprocess.run(shlex.split(find_command), stdout=subprocess.PIPE, stderr=subprocess.PIPE,
                                   check=True)
-            proc_lines = proc.stdout.decode().splitlines()
+            configs_json = json.loads(proc.stdout.decode())
+            for config in configs_json['configurations']:
+                if cfs_config_prefix in config['name']:
```

Comment:
We talked about using `startswith` instead of `in` to just be a little more sure we're only deleting the items created by the tests.

```suggestion
                if config['name'].startswith(cfs_config_prefix):
```